reference/5.1/Microsoft.PowerShell.Utility/Export-PSSession.md

```diff
@@ -67,12 +69,12 @@ This example exports all of the `Get` and `Set` commands from a server.
 
 ```powershell
 $S = New-PSSession -ConnectionUri http://exchange.microsoft.com/mailbox -Credential exchangeadmin01@hotmail.com -Authentication Negotiate
-Export-PSSession -Session $R -Module exch* -CommandName Get-*, Set-* -FormatTypeName * -OutputModule $pshome\Modules\Exchange -Encoding ASCII
+Export-PSSession -Session $S -Module exch* -CommandName Get-*, Set-* -FormatTypeName * -OutputModule $PSHOME\Modules\Exchange -Encoding ASCII
 ```
 
 These commands export the `Get` and `Set` commands from a Microsoft Exchange Server snap-in on a
-remote computer to an Exchange module in the `$pshome`\Modules directory on the local computer.
-Placing the module in the `$pshome`\Modules directory makes it accessible to all users of the
+remote computer to an Exchange module in the `$PSHOME`\Modules directory on the local computer.
+Placing the module in the `$PSHOME`\Modules directory makes it accessible to all users of the
```

Comment:
Let's change this to:
~~~
`$PSHOME\Modules`
~~~

Fix this in all versions.

Reply:
fixed